src/MenuItemTuning.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright © 2018-2023 Synthstrom Audible Limited
+ * Copyright © 2023 Casey Tucker
```

Comment:
I'd like some feedback regarding what is expected and appropriate regarding attribution. Now that this is a community project, there will be more authors, and while attribution can seen in git commit messages, maybe when new files are introduced or heavily modified we would want to include authors in the copyright notices. Adding my name below the existing attributions seemed to be the recommended way after a cursory web search, but I'd like to know what the maintainers think.

Reply:
I think attribution makes sense, but Copyright is tricky territory in open source and I would not accept any PRs with a unique license term or copyright without running it by the Synthstrom team.

The bulk of the community source code is, as I understand it, *Copy-left* and licensed under the GNU General Public License with some pieces being under license from Renesas or available under the public domain. I would consider a "copyright" notice from Synthstrom of this sort to be overridden by the General Public License, effective after June 5, 2023 but this is probably territory for @ian-jorgensen to weigh in on.